docs/core/Graph.md

```diff
@@ -31,22 +31,24 @@ Graph is a set of graphs that share a common set of base visualization structure
+| ---------------------- | -------- | ---------------------------------------------- | ------------------------------------------------------------------------------------ |
+| bindLegendTo           | string   | null                                           | If DOM id provided, binds legend into that container (Example: `"#legendContainer"`) |
+| locale                 | object   | LOCALE.en_US                                   | Locale object for X-Axis tick values                                                 |
+| dimension              | object   | {}                                             | Refer [Dimension](#dimension)                                                        |
+| throttle               | number   | (1000/60) => time in ms                        | Delay between resize of a browser window                                             |
+| showLabel              | boolean  | true                                           | Toggle to show axes labels                                                           |
+| showLegend             | boolean  | true                                           | Toggle to show graph legend                                                          |
+| showShapes             | boolean  | true                                           | Toggle to show shapes in the line graph                                              |
+| showVGrid              | boolean  | true                                           | Toggle to show vertical grid                                                         |
+| showHGrid              | boolean  | true                                           | Toggle to show horizontal grid                                                       |
+| dateline               | array    | []                                             | Refer [Dateline](../helpers/Dateline.md)                                             |
+| pan                    | object   | {}                                             | Refer [Panning](../controls/Panning.md)                                              |
+| padding                | object   | `{ top: 10, bottom: 10, left: 30, right: 50 }` | Refer [Padding](../controls/Padding.md)                                              |
+| showNoDataText         | boolean  | true                                           | Toggle to show no data text                                                          |
+| allowCalibration       | boolean  | true                                           | Toggle to allow calibration to adjust the y axis                                     |
+| removeContainerPadding | boolean  | false                                          | Toggle to remove carbon container top and bottom padding                             |
+| legendPadding          | object   | `{ top: 4, bottom: 4, left: 8, right: 8 }`     | Sets the legend padding and removes the carbon default margin of 8px around legend   |
```

Comment:
Nit: Alphabetize

Reply:
Alphabetized the props [here](https://github.com/JananiGunasekaran/carbon-graphs/commit/751bb4e91a16e2653e7d094c18ae089e4393678f).